Update error message to include link to readme.

ailzhang opened the pull request:
Test Plan: CI

Differential Revision: D24430852

ezyang commented on tools/codegen/gen.py:
```diff
@@ -763,6 +763,9 @@ def compute_declaration_yaml(f: NativeFunction) -> object:
         and Variant.method not in f.variants
 
+    # Having only Math in dispatch section is equivalent to no dispatch section.
+    is_abstract = f.dispatch is not None and not (len(f.dispatch) == 1 and 'Math' in f.dispatch)  # type ignore
```
You could probably just do a direct equality test on the key set here

Commit:
Update error message to include link to readme. (#46613)

Summary: Pull Request resolved: https://github.com/pytorch/pytorch/pull/46613

Test Plan: CI

Reviewed By: ezyang

Differential Revision: D24430852

fbshipit-source-id: ea2c8162c898a584b0ca84f141d26e9db108749b
